scripts/manage-members.js

```diff
@@ -0,0 +1,177 @@
+'use strict'
+
+const P = require('effd')
+const request = require('request')
+const debug = require('debug')('manage-members')
+const logger = require('../lib/logger.js')
+
+var login
+const github = require('../lib/github-client.js')
+// store/cache info about the authenticated user
+github.users.get({}).then((user) => {
+  debug('authenticated as:', user.login)
+  login = user.login
+})
+.catch((err) => {
+  logger.error('authentication failed', err)
+})
+
+// we need this RegExp to have the global flag
+const mentionsRegex = new RegExp(require('mentions-regex')().source, 'g')
+const exclude = (a, b) => a.filter((x) => !b.find((y) => y === x))
+const README = (file) => /^readme\.md$/i.test(file.filename || file) // array of strings or file objects
+
+const ignoredErrors = new Set([
+  'Not Applicable',
+  'No members changed',
+  'Members section not found',
+  'Push is on non-default branch'
+])
+const logError = (err) => {
+  if (ignoredErrors.has(err && err.message)) {
+    return debug(err.message)
+  }
+  logger.error(err)
```

Comment:
It would be really nice if this could include more context. Rich logs has been really valuable when debugging bot issues so far.

At least a couple of things comes to mind: which repo and the related PR number